src/test/java/com/webank/webase/front/gm/pressure/PerformanceHelloWorldCallback.java

```diff
@@ -0,0 +1,39 @@
+package com.webank.webase.front.gm.pressure;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.fisco.bcos.channel.client.TransactionSucCallback;
+import org.fisco.bcos.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PerformanceHelloWorldCallback extends TransactionSucCallback {
+  private static ObjectMapper objectMapper = new ObjectMapper();
+  private Long startTime = System.currentTimeMillis();
+
+  private PerformanceCollector collector;
+
+  public PerformanceCollector getCollector() { return collector; }
+
+  public void setCollector(PerformanceCollector collector) {
+    this.collector = collector;
+  }
+
+  static Logger logger = LoggerFactory.getLogger(PerformanceHelloWorldCallback.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)